src/main/java/frc/robot/subsystems/PivotSubsystem.java

```diff
@@ -0,0 +1,56 @@
+package frc.robot.subsystems;
```

Comment:
It looks like this file is already in https://github.com/Mechanical-Mustaches/2024-robot/pull/8. Remember to branch off of main when making different changes.

Reply:
whoops :O